README.md

```diff
@@ -1,70 +1,183 @@
+### Prerequisites
 
-### Making a Progressive Web App
+In order to run this project you need Webpack deveopment environment installed for this on you local machine
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Setup
 
-### Advanced Configuration
+Clone this repository to your desired folder on your local machine
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+### Install
+
+The projects incorporate some linters check, to get them running on your local system follow these below;
+
+1. `.github/workflows/linters.yml` is already included in project, this is used by Github to run tests on the repo.
+2. To install Webpack packages and dependencies
+
+- `npm init -y` initalize the project.
+- `npm i` installs all dependencies listed in the projects's package.json
+
+- `npm install webpack webpack-cli --save-dev` installs webpack
+- `npm install --save-dev html-webpack-plugin` installs webpack plugin
+- `npm install --save-dev style-loader css-loader` installs assets loaders
+- `npm install --save-dev webpack-dev-server` installs webpack server
+
+3. To install Stylelint linters run:
+
+- `npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`
+- `npx stylelint "\*_/_.{css,scss}" --fix" to run the test.
+
+4. To Install ESLint run:
+
+- `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x`
+- `npx eslint . --fix` to run the test
+
+5. To install Webhint linters run:
+
+- `npm install --save-dev hint@7.x`
+- `npx hint .` to run the test.
+
+### Usage
+
+To run the project, clone the repository to your local machine, install dependencies and launch through webpack Server on VScode
+
+### Run tests
+
+Follow the procedures under [Install](#install) to run some of the included linter tests.
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+- [)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Moses-Chibuike**
+
+- GitHub: [@githubhandle](https://github.com/Moses-chibuike)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ezechukwu-chibuike/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcomed!
+
+Feel free to check the [issues page]().
```

Comment:
- [ ] This issue link is broken kindly make sure to include the correct link here so it redirect to the issue page